SupersonicSound/Wrapper/Fmod/fmod.cs

```diff
@@ -16,8 +16,8 @@ namespace FMOD
     */
     public class VERSION
     {
-        public const int    number = 0x00010700;
```

Comment:
I assume all the changes in this file were just copied from the FMOD C# wrapper, and were not written by you? If so that's fine :)